### ISSUE TITLE
How to get MLM outputs?

### ISSUE THREAD

**Issue by sharavsambuu** [kpe/bert-for-tf2#52]
Hello, great library!
How to return vocabulary sized logits used in Masked Language Model instead of embeddings? Any examples?
